doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc

```diff
@@ -22,8 +22,12 @@ Both Infinispan and Keycloak are deployed in the same namespace.
 
 . Check out the Git repository from https://github.com/keycloak/keycloak-benchmark/.
 
+. Create a Route53 entry as described in xref:./installation-route53-loadbalancer.adoc[].
```

Comment:
This document details how to use the various *.sh scripts directly, but we now have Tasks for this which greatly simplify things. Should we update this document as well? I'm happy for this to be done as a separate issue.

Reply:
I wasn't aware of the taskfile integration. Please create a follow-up issue for this one.

If everything else is fine, please approve and merge. Thanks!

Reply:
I've created https://github.com/keycloak/keycloak-benchmark/issues/826 to track the work.